docs/gallery.rst

```diff
@@ -182,95 +182,69 @@ Clustering Visualizers
     :width: 200px
     :height: 100px
-    :alt: Learning Curve for MultinomialNB
-    :target: api/model_selection/learning_curve.html#classification
+    :alt: Learning Curves for Data Sufficiency
+    :target: api/model_selection/learning_curve.html
 
-.. image:: api/model_selection/images/learning_curve_clusterer.png
+.. image:: images/gallery/cv_scores.png
     :width: 200px
     :height: 100px
-    :alt: Learning Curve for KMeans
-    :target: api/model_selection/learning_curve.html#clustering
+    :alt: Cross Validation Scores
+    :target: api/model_selection/cross_validation.html
 
-.. image:: api/model_selection/images/cv_scores_classifier.png
+.. image:: images/gallery/decision_boundaries.png
     :width: 200px
     :height: 100px
-    :alt: CV Scores for MultinomialNB Classification
-    :target: api/model_selection/cross_validation.html#classification
+    :alt: Nearest Neighbor Decision Boundary Prototype
+    :target: api/contrib/boundaries.html#decisionboundaries-vizualizer
 
-.. image:: api/model_selection/images/cv_scores_regressor.png
-    :width: 200px
-    :height: 100px
-    :alt: CV Scores for Ridge Regression
-    :target: api/model_selection/cross_validation.html#regression
 
 Text Modeling Visualizers
 ---------------------------
 
-.. image:: api/text/images/freqdist_corpus.png
+.. image:: images/gallery/freqdist.png
     :width: 200px
     :height: 100px
-    :alt: Validation Curve for Decision Tree Regresor
+    :alt: Token Frequency Distribution
     :target: api/text/freqdist.html#token-frequency-distribution
 
-.. image:: api/text/images/tsne_all_docs.png
+.. image:: images/gallery/corpus_tsne.png
     :width: 200px
     :height: 100px
     :alt: TSNE Projection of Documents
     :target: api/text/tsne.html#t-sne-corpus-visualization
 
-.. image:: api/text/images/dispersion_docs.png
+.. image:: images/gallery/dispersion.png
     :width: 200px
     :height: 100px
-    :alt: Dispersion Plot
+    :alt: Dispersion of Words in a Corpus
     :target: api/text/dispersion.html#dispersion-plot
 
-Decision Boundaries Visualizer
-------------------------------
-
-.. image:: api/contrib/images/knn_decisionviz.png
-    :width: 200px
-    :height: 100px
-    :alt: Nearest Neighbor Boundary Visualizer
-    :target: api/contrib/boundaries.html#decisionboundaries-vizualizer
-
 Target Visualizers
 ------------------
 
-.. image:: api/target/images/balanced_binning_reference.png
+.. image:: images/gallery/balanced_binning_reference.png
     :width: 200px
     :height: 100px
     :alt: Balanced Binning Reference
     :target: api/target/binning.html#balanced-binning-reference
 
-.. image:: api/target/images/class_balance_compare.png
+.. image:: images/gallery/class_balance.png
     :width: 200px
     :height: 100px
     :alt: Class Balance
     :target: api/target/class_balance.html#class-balance
 
-.. image:: api/target/images/feature_correlation_pearson.png
+.. image:: images/gallery/feature_correlation.png
```

Comment:
any particular reason to remove other feature_correlation images

Reply:
They just seemed kind of redundant and didn't add anything new to the gallery; it was just three bar charts next to each other. I wanted the gallery to feel like a high-level overview of all the things YB could do.

Reply:
noted!